tests/phpunit/elementor/core/experiments/test-manager.php

```diff
@@ -44,6 +45,52 @@ public function reset_experiments_state( Experiments_Manager $experiments_manage
 		$experiments_manager->set_feature_default_state( $experimental_data['name'], $original_experiment_state );
 	}
 
+	public function test_features_dependencies_initial_state_mismatches() {
+		new \Elementor\Core\Modules_Manager();
+		$features = \Elementor\Plugin::$instance->experiments->get_features();
+		$this->assertNotEmpty( $features );
+		foreach ( $features as $feat ) {
+			if ( ! isset( $feat['dependencies'] ) ) {
```

Comment:
please reduce the nesting level ( extract to private functions ) and make it more readable